sdk/cosmosdb/cosmos/src/client/Item/Items.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
-import { v4 as uuid } from "uuid";
+import { v4 } from "uuid";
+const uuid = v4;
```

Comment:
why were these small changes needed?

Reply:
I had enabed the linter for samples and these were complaining, however will let me know we don't typically lint these samples files so I turned it off but left the changes

Reply:
not sure why it suddenly began complaining on these now